frontend/package.json

```diff
@@ -70,7 +73,7 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.35",
     "@fortawesome/free-solid-svg-icons": "^5.15.3",
     "@juggle/resize-observer": "^3.3.1",
-    "@mempool/mempool.js": "^2.2.4",
+    "@mempool/mempool.js": "2.2.6",
```

Comment:
This should be v2.3.0-dev1 etc. until the final release of v2.3.0 so it matches the backend.